libs/ledger/include/ledger/storage_unit/storage_unit_client.hpp

```diff
@@ -38,6 +38,7 @@
 #include "storage/document_store_protocol.hpp"
 #include "storage/object_store_protocol.hpp"
 
+#include <array>
```

Comment:
This file doesn't seem to use `std:array`. You could include `<cassert>` and also `<cstring>` for `std::memcpy`.

Reply:
`std::array` is used in the `MerkleTreeBlock` (this will hopefully be updated soon). I have added those header files